apps/frontend/package.json

```diff
@@ -3,7 +3,7 @@
     "version": "0.0.0",
     "type": "module",
     "scripts": {
-        "dev": "vite",
+        "dev": "vite --host",
```

Comment:
It's better to add two different scripts. One with, and one without exposition. Because not everyone (me for example) wants to expose, ha ha.